deluge/common.py

```diff
@@ -730,34 +730,41 @@ def is_ip(ip):
     """
     A simple test to see if 'ip' is valid
```

Comment:
Docstrings should start on the first line as opening quotes: `"""A simple test...`